bin/producer-build-image.sh

```diff
@@ -1,26 +0,0 @@
-#!/bin/bash
-
-#
-# Will use docker to build the necessary jar, then extract the jar and add it to
-# a clean image which only contains Java.
-#
-
-set -e
-
-TAG=${1:latest}
-
-echo "========== BUILDING BASE IMAGE ========="
-BASE_ID="pkolloch/tweet-producer-base"
-docker build -t "$BASE_ID" .
-
-echo "========== PREPARE PRODUCTION CONTAINER ========="
-mkdir -p twitter/target/docker
-cp Dockerfile.tweet-producer twitter/target/docker/Dockerfile
-cd twitter/target/docker
-docker run --rm=true "$BASE_ID" cat /twitter-assembly-1.0.jar >twitter-assembly-1.0.jar
```

Comment:
BTW, I did this because I had production issues with the "full-size" images. They were so big that I saw cache corruption issues with docker (maybe because of some size limits or because bigger images just made the error more likely).